packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -577,28 +581,47 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
     ]
   );
 
-  const onRenderChatContent = useCallback(
-    (): JSX.Element => (
+  const onRenderChatContent = useCallback((): JSX.Element => {
+    /* @conditional-compile-remove(breakout-rooms) */
+    if (!isChatInitialized) {
+      return (
+        <Stack styles={chatSpinnerContainerStyles}>
+          <Spinner size={SpinnerSize.large} />
```

Comment:
Any a11y requirements to state why a spinner is showing similar to an alt text for image?

Reply:
For this spinner is there any label information to add like "Fetching Chat Thread" or something?

Reply:
I will add the label Teams has
![image](https://github.com/user-attachments/assets/c2d413da-eab7-4d9f-9dd3-9d26b4dc1f54)
and this is what it looks like in our composite
![image](https://github.com/user-attachments/assets/51baa3b8-4a81-413e-8c28-5f592bd50590)